### PR TITLE
Add game_of_clouds.md

### PR DESCRIPTION
**Pull request by rishabmenon**
Added a comical new haiku called "game_of_clouds.md" based on cloud competitors. 
It's called "game_of_clouds"!

### CODE REVIEW

File: _haikus/game_of_clouds.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+layout: haiku
+title: Game of Clouds
+author: Rishab Menon
+---
+
+Iron Throne win who? <br>
+Google, Microsoft, AWS? <br>
```
Comment (ltagliaferri) [inline _haikus/game_of_clouds.md]: This line has one extra syllable, maybe you can sub `Azure` for Microsoft

File: _haikus/game_of_clouds.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+Amazon it is! <br>
```
Comment (ltagliaferri) [inline _haikus/game_of_clouds.md]: No love for DigitalOcean? ;)

File: _haikus/game_of_clouds.md (hunk 1 above)
Comment (rishabmenon) [reply _haikus/game_of_clouds.md]: Yes, sure thing! Thanks for the suggestion :)

File: _haikus/game_of_clouds.md (hunk 2 above)
Comment (rishabmenon) [reply _haikus/game_of_clouds.md]: hahah..couldn't fit into it, or you know who is always the winner! ;)

**Comment by beardicus** [comment]
Thanks! Merged. The failed tests were for a broken haiku elsewhere in the build, now fixed.

**Pull request merged**